wavefront_pyformance/wavefront_reporter.py

```diff
@@ -185,9 +188,14 @@ def __init__(self, server, token, source='wavefront-pyformance',
         self.server = self._validate_url(server)
         self.token = token
         self.batch_size = 10000
-        self.wavefront_client = wavefront_sdk.WavefrontDirectClient(
-            self.server, token, batch_size=self.batch_size,
+
+        client_factory = WavefrontClientFactory()
+        client_factory.add_client(
+            url="https://{}@{}".format(token, urlparse(server).hostname),
```

Comment:
Actually we probably shouldn't parse to put only hostname. There's case such as use Direct ingestion to send to "localhost:8080", dropping the port will fail such usecase